commands/translate.py

```diff
@@ -0,0 +1,169 @@
+from commands.base import Command
+from discord import Embed
+from helpers import *
+from utils.embed_table import EmbedTable
+
+import googletrans
+import json
+import re
+
+HISTORY = 10
+HISTORY_LIMIT = 100
+FLAGS_FILE = "files/flags.json"
+AVATAR_FORMAT = "https://cdn.discordapp.com/avatars/{0.id}/{0.avatar}.png?size=128"
+TRANSLATE_COLOR = int('4885ed', 16)
+
+class Translate(Command):
+    desc = """Translates a marked message to another language (removes all emojis).
+    React with a flag and type `!translate` to translate the message to the corresponding language.
+    Optional `index` argument defines offset of message to translate in list of marked messages."""
+
+    async def eval(self, index=0):
+        if int(index) < 0:
+            raise CommandFailure('Invalid index!')
+
+        archive = await create_list(self.client.logs_from(self.message.channel,
+                                                            limit=HISTORY_LIMIT))
+
+        if not archive:
+            raise CommandFailure('No tagged messages in the last %s found!' % HISTORY_LIMIT)
+
+        if int(index) >= len(archive):
+            raise CommandFailure('Invalid index!')
+
+        lang_list = archive[int(index)]
+        for entry in lang_list:
+            #translate contents
+            translator = googletrans.Translator()
```

Comment:
This will create an instance of Translator() for every message to be translated - is this necessary or is only one instance needed?